to_copy/run.sh

```diff
@@ -400,6 +359,33 @@ rem "Configuring and starting strongSwan"
     encryptme-stats --server "$ENCRYPTME_STATS_SERVER" $ENCRYPTME_STATS_ARGS &
 }
 
+
```

Comment:
- ensure this logic is refactored to work in what I provided in my latest changes
- instead of managing a temp file w/ rules, far better to use the pep-filter.sh script to reinitialize state

Reply:
Logic to reinitiate state is now in pep-filter.sh, and called from run.sh